src/MultiMessageSender.sol

```diff
@@ -59,7 +59,7 @@ contract MultiMessageSender {
     /// @dev checks if msg.sender is caller configured in the constructor
     modifier onlyCaller() {
         if (msg.sender != caller) {
-            revert Error.INVALID_PREVILAGED_CALLER();
+            revert Error.INVALID_PRIVILAGED_CALLER();
```

Comment:
nit: "privilaged" -> "privileged" (there are a number of these throughout the PR)